builds/jazz-build-module/config-loader.groovy

```diff
@@ -11,24 +12,84 @@ import groovy.transform.Field
 
 echo "the module, 'config-loader', loaded successfully... congratulations..."
 
-def initialize(installerVars){
-  if(installerVars != null){
-		def resultJson = parseJson(installerVars)
+/**
+ * Load the service metadata from Catalog
```

Comment:
Fix the comment. This method loads Jazz system `config` and not `service metadata` (which sounds more like a specific service's metadata).

Reply:
done